101-lustre-client-server-create/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# Lustre client and server node VMs deployment
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-lustre-client-server-create/azuredeploy.json" target="_blank">
+<img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-lustre-client-server-create/azuredeploy.json" target="_blank">
+<img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+
+This template creates Lustre client and server node VMs and related infrastructure such as VNETs. Provided below are the relevant details of the deployment.
```

Comment:
@lustrehpc Interesting! Thanks for finding the history! What do I need to deploy before deploying this template?